// @vonage/voice/dist/enums/TTSLanguages.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.TTSLanguages = void 0;
/**
 * Enum representing the supported Text-to-Speech (TTS) languages for Nexmo's voice calls.
 */
var TTSLanguages;
(function (TTSLanguages) {
    /**
     * Arabic (ar) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["AR"] = "ar";
    /**
     * Catalan (ca-ES) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["CA_ES"] = "ca-ES";
    /**
     * Mandarin Chinese (cmn-CN) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["CMN_CN"] = "cmn-CN";
    /**
     * Mandarin Chinese (cmn-TW) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["CMN_TW"] = "cmn-TW";
    /**
     * Czech (cs-CZ) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["CS_CZ"] = "cs-CZ";
    /**
     * Welsh (cy-GB) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["CY_GB"] = "cy-GB";
    /**
     * Danish (da-DK) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["DA_DK"] = "da-DK";
    /**
     * German (de-DE) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["DE_DE"] = "de-DE";
    /**
     * Greek (el-GR) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["EL_GR"] = "el-GR";
    /**
     * Australian English (en-AU) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["EN_AU"] = "en-AU";
    /**
     * British English (en-GB) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["EN_GB"] = "en-GB";
    /**
     * British English (en-GB-WLS) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["EN_GB_WLS"] = "en-GB-WLS";
    /**
     * Indian English (en-IN) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["EN_IN"] = "en-IN";
    /**
     * US English (en-US) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["EN_US"] = "en-US";
    /**
     * South African English (en-ZA) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["EN_ZA"] = "en-ZA";
    /**
     * Spanish (es-ES) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["ES_ES"] = "es-ES";
    /**
     * Mexican Spanish (es-MX) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["ES_MX"] = "es-MX";
    /**
     * US Spanish (es-US) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["ES_US"] = "es-US";
    /**
     * Basque (eu-ES) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["EU_ES"] = "eu-ES";
    /**
     * Finnish (fi-FI) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["FI_FI"] = "fi-FI";
    /**
     * Filipino (fil-PH) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["FIL_PH"] = "fil-PH";
    /**
     * Canadian French (fr-CA) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["FR_CA"] = "fr-CA";
    /**
     * French (fr-FR) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["FR_FR"] = "fr-FR";
    /**
     * Hebrew (he-IL) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["HE_IL"] = "he-IL";
    /**
     * Hindi (hi-IN) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["HI_IN"] = "hi-IN";
    /**
     * Hungarian (hu-HU) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["HU_HU"] = "hu-HU";
    /**
     * Indonesian (id-ID) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["ID_ID"] = "id-ID";
    /**
     * Icelandic (is-IS) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["IS_IS"] = "is-IS";
    /**
     * Italian (it-IT) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["IT_IT"] = "it-IT";
    /**
     * Japanese (ja-JP) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["JA_JP"] = "ja-JP";
    /**
     * Korean (ko-KR) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["KO_KR"] = "ko-KR";
    /**
     * Norwegian Bokmål (nb-NO) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["NB_NO"] = "nb-NO";
    /**
     * Dutch (nl-NL) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["NL_NL"] = "nl-NL";
    /**
     * Norwegian (no-NO) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["NO_NO"] = "no-NO";
    /**
     * Polish (pl-PL) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["PL_PL"] = "pl-PL";
    /**
     * Brazilian Portuguese (pt-BR) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["PT_BR"] = "pt-BR";
    /**
     * Portuguese (pt-PT) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["PT_PT"] = "pt-PT";
    /**
     * Romanian (ro-RO) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["RO_RO"] = "ro-RO";
    /**
     * Russian (ru-RU) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["RU_RU"] = "ru-RU";
    /**
     * Slovak (sk-SK) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["SK_SK"] = "sk-SK";
    /**
     * Swedish (sv-SE) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["SV_SE"] = "sv-SE";
    /**
     * Thai (th-TH) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["TH_TH"] = "th-TH";
    /**
     * Turkish (tr-TR) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["TR_TR"] = "tr-TR";
    /**
     * Ukrainian (uk-UA) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["UK_UA"] = "uk-UA";
    /**
     * Vietnamese (vi-VN) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["VI_VN"] = "vi-VN";
    /**
     * Cantonese Chinese (yue-CN) - Supported Text-to-Speech (TTS) language.
     */
    TTSLanguages["YUE_CN"] = "yue-CN";
})(TTSLanguages || (exports.TTSLanguages = TTSLanguages = {}));
